src/index.ts

```diff
@@ -59,15 +69,25 @@ const client = new Bot({
 });
 
 // Listener for the on ready event
-client.once(Events.ClientReady, (c: Client): void => {
+client.once(Events.ClientReady, async (c: Client): Promise<void> => {
+    console.log("Fetching the user reports channel...");
+    const userReports: Channel = getUserReportsChannel(c);
+    console.log(`Found ${userReports}`);
+
+    console.log("Fetching the error logs channel...");
+    const logs: Channel = getAdminLogsChannel(c);
+    console.log(`Found ${logs}`);
+
+    console.log("Running initialisation routines...");
+    await refreshBlacklist(c);
+    console.log("Done initialising!");
+    
```

Comment:
The startup of the bot has been changed to check for certain channels and running initialisation routines

Currently, it fetches 2 channels and refreshed the blacklist